heudiconv/utils.py

```diff
@@ -103,9 +103,22 @@ def dec(obj):
 
 
 def anonymize_sid(sid, anon_sid_cmd):
+    import sys
     from subprocess import check_output
+    
     cmd = [anon_sid_cmd, sid]
-    return check_output(cmd).strip()
+    shell_return = check_output(cmd)
```

Comment:
How about condensing this into:
```suggestion
   kwargs = {'encoding': 'utf-8'} if sys.version_info[0] >= 3 else {}
   return check_output(cmd, **kwargs).strip()
```

Reply:
definitely better ++

Reply:
looks like `encoding` wasn't added to `check_output` til 3.6 😓 - for the sake of backwards compatibility let's go with your original approach (decoding after subprocess call)